web/src/app/browser/build.sh

```diff
@@ -13,6 +13,7 @@ SUBPROJECT_NAME=app/browser
 # ################################ Main script ################################
 
 builder_describe "Builds the Keyman Engine for Web's website-integrating version for use in non-puppeted browsers." \
+  "@/common/web/types build" \
```

Comment:
If this has already been added by `@/common/web/keyboard-processor`, including it here is redundant.  That's why there's no `@/common/web/keyboard-processor` listed in the original version - that, in turn, is auto-included via `input-processor`, then `engine/main` - the last of which _is_ listed.

Granted, the fact that we _are_ directly importing from it in a few files may be worth keeping it listed, I guess.

```suggestion
```